src/test/java/jalau/at18/architects/ScienceComparatorTest.java

```diff
@@ -0,0 +1,38 @@
+package jalau.at18.architects;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.LinkedList;
+
+import org.junit.Test;
+
+public class ScienceComparatorTest {
+    @Test
+    public void shouldHaveTheSameElement(){
+        GreenCard card1 = new GreenCard(ScientificSymbol.SCREW);
+        GreenCard card2 = new GreenCard(ScientificSymbol.SCREW);
+        LinkedList<String> listOfScienceElements = new LinkedList<String>();
+        ScienceCounter scienceCounter = new ScienceCounter(listOfScienceElements);
```

Comment:
Is it expected to have a case where we send a non-empty list to the constructor? if not, I think this list can be initialized inside the constructor of `ScienceCounter` without needing to send a parameter